IMC_Python/Storing_Data_In_Python/Storing_Data_in_Python.md

```diff
@@ -0,0 +1,253 @@
+# Storing Data in Python
+
+When we think about Python, we can think of tons of applications for our potential code; we can create a desktop application, build websites, analyze data, and much more - the possibilities are endless. However, in order to do anything of that caliber with this programming language, you must start from the bottom and master the basics. One essential building block of Python is learning to store pieces of data that will be useful to our program and, more importantly, deciding how we will store this data. Lists, sets, and dictionaries are the most common ways to store data and, in this blog post, I will show you how you can use them, why you would want to use them, and what features each data structure has to offer. 
```

Comment:
Try to avoid gerunds (-ing verbs) to make writing more clear. I also cut up some sentences with periods. The variety of sentence structure is nice, just beware comma usage.